pkg/notify/notify.go

```diff
@@ -172,67 +173,68 @@ func toIssueLabel(groupLabels alertmanager.KV) string {
 	return strings.Replace(buf.String(), " ", "", -1)
 }
 
-func (r *Receiver) search(project, issueLabel string) (*jira.Issue, bool, error) {
+func (r *Receiver) search(project, issueLabel string, logger log.Logger) (*jira.Issue, bool, error) {
 	query := fmt.Sprintf("project=\"%s\" and labels=%q order by resolutiondate desc", project, issueLabel)
 	options := &jira.SearchOptions{
 		Fields:     []string{"summary", "status", "resolution", "resolutiondate"},
 		MaxResults: 2,
 	}
-	log.V(1).Infof("search: query=%v options=%+v", query, options)
+	level.Debug(logger).Log("msg", "search", "query", query, "options", options)
 	issues, resp, err := r.client.Issue.Search(query, options)
 	if err != nil {
-		retry, err := handleJiraError("Issue.Search", resp, err)
+		retry, err := handleJiraError("Issue.Search", resp, err, logger)
 		return nil, retry, err
 	}
 	if len(issues) > 0 {
 		if len(issues) > 1 {
 			// Swallow it, but log a message.
-			log.Infof("More than one issue matched %s, will only update last issue: %+v", query, issues)
+			level.Warn(logger).Log("msg", "more than one issue matched, updating only the last one", "query", query, "issues", issues)
```

Comment:
I was going to suggest rewording this, for the same reason as above: we now have a `reopen_duration`, which may mean we don't actually update the most recent issue.

But the `reopen_duration` is all but guaranteed to result in multiple issues with the same label being created over time, so I'd say this warning should be dropped entirely. Or on second (now third) thought, demoted to debug. And slightly reworded.
```suggestion
			level.Warn(logger).Log("msg", "more than one issue matched, picking most recently resolved", "query", query, "issues", issues)
```